pkg/appconsts/consensus_consts.go

```diff
@@ -9,5 +9,5 @@ const (
 	// interval isn't enforced at consensus, the real block interval isn't
 	// guaranteed to exactly match GoalBlockTime. GoalBlockTime is currently targeted
 	// through static timeouts (i.e. TimeoutPropose, TimeoutCommit).
-	GoalBlockTime = time.Second * 15
+	GoalBlockTime = time.Second * 6
```

Comment:
I'm curious if this will cause block time to be less than 6s? As before it was 15, but block times average around 12s on all networks.

Reply:
Decreasing the value for this constant won't actually lower block times. It looks like this constant is used in 3 places:

1. To determine Mempool TTLDuration
2. To determine Evidence MaxAgeNumBlocks
3. To determine x/ibc MaxExpectedTimePerBlock

Reply:
thank you for following up here. I think then it may be safe to close this PR?

Reply:
yea we re-evaluate what those values should be. It doesn't have to strictly happen in #3882 but let's create a follow-up to tackle that. Thanks!

Reply:
sweet, thanks for the help and context.